palak/src/components/PopupColors.tsx

```diff
@@ -0,0 +1,23 @@
+import Popup from "../popup/Popup";
+
+interface ColorProps {
+  bgColor: string;
+  glassBgColor: string;
+  textColor: string;
+  inputBgColor: string;
+  buttonBgColor: string;
+  spanColor: string;
+  popupBgColor: string;
```

Comment:
are all these values really necessary  for the components? Can we reduce few props like buttonBgColor, InputBGColor as they are their own components and should have default values for their variants